docs/how-to/create-project.md

```diff
@@ -0,0 +1,35 @@
+# Create a Project
```

Comment:
The screenshots for this doc are not showing up at the deployed URL. Could you confirm this is working?

Reply:
Strange, it was working just now. it's not working for me now as well. let me check

Reply:
fixed this.